app/PlatformVerifier/Env.php

```diff
@@ -22,25 +22,46 @@ class Env
                 "See https://laravel.com/docs/5.8/queues for more information on queue drivers.",
     ];
 
-    public static function verifyRequirements()
+    public static function verifyRequirements($console = true)
     {
+        $ok = "Good job! you have configured your .ENV file with all the required keys.";
+        $info = "We will check the database connectivity next.";
```

Comment:
@rowasc Can we remove "We will check the database connectivity next" from the response? It gets a bit confusing (depending on the checks and promises, it might not print the db-check next). Its ok to keep in in the console print.

Reply:
ok. 👍

Reply:
I left it in the CLI and removed it from the endpoint's output :)